pipeline/07-export.R

```diff
@@ -413,14 +454,35 @@ assessment_card_prepped <- assessment_card %>%
     township_code, meta_pin, meta_card_num, meta_class, meta_nbhd_code,
     meta_card_pct_total_fmv, pred_card_initial_fmv, pred_card_final_fmv,
     char_yrblt, char_beds, char_ext_wall, char_heat, char_bldg_sf,
-    char_type_resd, char_land_sf
+    char_type_resd, char_land_sf, char_apts, char_ncu
   ) %>%
   mutate(
     meta_pin = glue(
       '=HYPERLINK("https://www.cookcountyassessor.com/pin/{meta_pin}",
       "{meta_pin}")'
+    ),
+    # Make sure the format of char_apts matches the short format we used to
+    # generate assessment_pin_prepped
+    char_apts = case_when(
+      char_apts == "None" | is.na(char_apts) ~ "Missing",
+      char_apts == "Two" ~ "2",
+      char_apts == "Three" ~ "3",
+      char_apts == "Four" ~ "4",
+      char_apts == "Five" ~ "5",
+      char_apts == "Six" ~ "6",
+      TRUE ~ "Missing"
```

Comment:
I don't love the duplication here, but again, I didn't see a dedicated `ccao` function for this logic, and it's only repeated once so it didn't feel worth it to factor out a helper function.